tests/visual/grid.html

```diff
@@ -82,6 +82,192 @@
 
         </div>
 
+        <div class="section">
+            <kendo-grid class="k-widget k-grid k-grid-no-scrollbar" style="width: 800px;">
```

Comment:
I think that a better place for these tests would be the grid-editing test, as the grid visual test is getting pretty crowded.